docs/mddocs/Quickstart/llama3_llamacpp_ollama_quickstart.md

```diff
@@ -51,6 +51,7 @@ To use GPU acceleration, several environment variables are required or recommend
   ```bash
   source /opt/intel/oneapi/setvars.sh
   export SYCL_CACHE_PERSISTENT=1
+  export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
```

Comment:
I think this will have a negative impact the performance on kernel 6.5? @hkvision

Reply:
Previously on workstation this will hinder the performance. Maybe do something similar to here? https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/dev/benchmark/all-in-one/run-arc.sh#L6

Reply:
On my side, I don't observe performance regression on arc10 (which is kernel 6.5.0 now). Reversely, I found `export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1` help improve performance.

Reply:
the issue was previously observed on Xeon, our machines are all Core, which don't have this problem.

Reply:
We don't have Xeon+Arc machine now, may add more performance validation later.